packages/pipeline/test/fixtures/copper/api_v1_activity_types.parsed.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I understand the raw response data being JSON, but I think the parsed data should be TypeScript. This would give us the benefit of the compiler helping us out if a field is ever renamed or something in the data structure changes. As it currently stands, we are bypassing the type checker by starting with JSON and then performing a blind type assertion (e.g., `(opportunitiesData[0] as unknown) as CopperOpportunity`). This seems like it could lead to unexpected bugs down the road.

What do you think?

Reply:
I think that's a good idea. I tried it and already found some problems with the parser saving `null` instead of `undefined`. 👍